script/script.js

```diff
@@ -9,4 +9,27 @@ menu.addEventListener('click', () => {
 document.querySelectorAll('.tem').forEach((n) => n.addEventListener('click', () => {
   menu.classList.remove('active');
   nav.classList.remove('active');
-}));
\ No newline at end of file
+}));
+// popup cards javascript
+const workContainer = document.querySelector('.popup');
+const workBox = workContainer.querySelectorAll('.pop');
+
+document.querySelectorAll('.workContent .card').forEach((work) => {
+  work.onclick = () => {
+    workContainer.style.display = 'flex';
+    const name = work.getAttribute('data-name');
+    workBox.forEach((preview) => {
+      const target = preview.getAttribute('data-target');
+      if (name === target) {
+        preview.classList.add('active');
+      }
+    });
+  };
+});
+
```

Comment:
- [ ]  You should Your pop-up card should fit well on the screen so that it can show all the information from your array. Maybe you can even use the scroll bar in your styles to make all the data appear 

![image](https://user-images.githubusercontent.com/43877689/173615041-3db433bb-3086-4db6-a49f-da01009fb655.png)